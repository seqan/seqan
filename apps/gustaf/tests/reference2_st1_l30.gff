gi|1234567|	GUSTAF	deletion	351	770	.	+	.	ID=0;size=420;support=1;supportIds=read1reference2,;breakpoint=281
