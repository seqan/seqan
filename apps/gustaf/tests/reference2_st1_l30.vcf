##fileformat=VCFv4.1
##source=GUSTAF
##reference=adeno.fa
##reads=read_reference2.fa
##INFO=<ID=IMPRECISE,Number=0,Type=Flag,Description="Imprecise structural variation">
##INFO=<ID=END,Number=1,Type=Integer,Description="End position of the variant described in this record (for DUP last position of duplicated sequence)">
##INFO=<ID=SVLEN,Number=.,Type=Integer,Description="Difference in length between REF and ALT alleles">
##INFO=<ID=SVTYPE,Number=1,Type=String,Description="Type of structural variant">
##INFO=<ID=EVENT,Number=1,Type=String,Description="Event identifier for breakends.">
##INFO=<ID=TARGETPOS,Number=1,Type=String,Description="Target position for duplications (position before inserted sequence).">
##INFO=<ID=BM,Number=1,Type=Integer,Description="Breakpoint Mate: similar breakpoints have same ID">
##INFO=<ID=DP,Number=1,Type=Integer,Description="Number of Supporting Reads/Read Depth for Variant">
##ALT=<ID=INV,Description="Inversion">
##ALT=<ID=DUP,Description="Duplication">
##ALT=<ID=DUP:TANDEM,Description="Tandem Duplication">
##contig=<ID=gi|9632547|ref|NC_002077.1|,length=4718>
##contig=<ID=gi|1234567|,length=1148>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT
gi|1234567|	350	.	T	<DEL>	.	PASS	SVTYPE=DEL;END=770;SVLEN=-420;BM=0;DP=1	.	1
