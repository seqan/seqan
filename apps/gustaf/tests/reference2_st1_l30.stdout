read_reference2.fa
Database file   : adeno.fa
Query file      : read_reference2.fa

Thresholds:
  overlap threshold (oth)          : 0.5
  gap threshold (gth)              : 5
  initial gap threshold (ith)      : 15
Penalties:
  translocation penalty (tp)       : 5
  inversion penalty (ip)           : 5
  order penalty (op)               : 0
  required read support (st)       : 1
Loading query sequences... Loaded 1 query sequence.
done
Loading reference sequences... Loaded 2 database sequences.
Loaded db seq with length: 4718
Loaded db ID: gi|9632547|ref|NC_002077.1|
Loaded db seq with length: 1148
Loaded db ID: gi|1234567|
done
Calling STELLAR...
Stellar options:
Database file   : adeno.fa
Query file      : read_reference2.fa

User specified parameters:
  minimal match length             : 30
  maximal error rate (epsilon)     : 0.05
  maximal x-drop                   : 5
  search forward strand            : yes
  search reverse complement        : yes

  verification strategy            : exact
  maximal number of matches        : 50
  duplicate removal every          : 500

Calculated parameters:
  k-mer length: 13
  s^min       : 13
  threshold   : 2
  distance cut: 40
  delta       : 16
  overlap     : 2

Constructing index...

TIME stellar 0.31517s
done
Getting match distance...
Constructing graphs... done
Analyzing graphs... done
 completed gff writing reference2_st1_l30.gff
 completed vcf writing reference2_st1_l30.vcf
TIME all 1.81999s
